code/drasil-data/Data/Drasil/Theories/Physics.hs

```diff
@@ -105,6 +106,27 @@ torqueDesc = foldlSent [S "The", phrase torque,
   S "to rotate the body around an axis or pivot"]
 
 --
+newtonSLR :: TheoryModel
+newtonSLR = tmNoRefs (cw newtonSLRRC)
+  [qw QP.torque, qw QP.momentOfInertia, qw QP.angularAccel] 
+  ([] :: [ConceptChunk]) [] [newtonSLRRel] [] "NewtonSecLawRotMot" newtonSLRNotes
+
+newtonSLRRC :: RelationConcept
+newtonSLRRC = makeRC "newtonSLRRC" 
+  (nounPhraseSP "Newton's second law for rotational motion") EmptyS newtonSLRRel
+
+newtonSLRRel :: Relation
+newtonSLRRel = sy QP.torque $= sy QP.momentOfInertia * sy QP.angularAccel
+
+newtonSLRNotes :: [Sentence]
+newtonSLRNotes = map foldlSent [
+  [S "The net", getTandS QP.torque, S "on a", phrase rigidBody `sIs`
+   S "proportional to its", getTandS QP.angularAccel `sC` S "where",
+   ch QP.momentOfInertia, S "denotes", phrase QP.momentOfInertia `ofThe`
+   phrase rigidBody, S "as the", phrase constant `sOf` S "proportionality"],
+  [S "We also assume that all", plural rigidBody, S "involved" `sAre`
```

Comment:
Can you double-check this with @smiths ? I think that this 2D assumption was correct in the context of Pendulum, but that it is actually not embedded in this law as stated here, i.e. the law here is general.

Reply:
Okay I see what you mean. Should I take the 2D bit out? I think it still makes sense in the context of the pendulum without that specification. Otherwise, is there a way to just mention that more specifically in `DblPendulum/Body.hs` by appending that sentence to a generalized version of `newtonSLR`?

Reply:
In some ways, the 2D note is redundant information, as it is already in the assumptions. And it will probably be in the instance model too. So since the theory model is supposed to be general, it makes sense to not have it at all.

If it should still be there (and hopefully @smiths will tell us), then the thing to do is to create a new theory model based on the generic one [using lenses to set the id field and the note field to be different].

Reply:
Okay, sounds good.

Reply:
Yes, we can take the 2D part out.  The equation looks exactly the same in 3D, so it is best if we keep the theory as general (and thus reusable) as possible.